TabGroup/tests/TabGroup-specs.js

```diff
@@ -3,6 +3,7 @@ import {render, screen} from '@testing-library/react';
 
```

Comment:
tabGroup's code coverage is already 100%, could you please add tests for onClick and icon (should render icon)? I wish you could get it from sandstone to save time.

Reply:
I added a test for rendering icons, but the test for onClick does not trigger the event :(
The Tab component has an onClick prop, but it doesn't appear to be called in the test :(
Here is the test that I tried:

	test('should fire `onClick` with `onClick` type when a tab is clicked', () => {
		const handleClick = jest.fn();
		render(
			<TabGroup
				tabPosition="before"
				tabs={[
					{title: 'Home', icon: 'home', onClick: handleClick},
					{title: 'Settings', icon: 'setting'},
					{title: 'Theme', icon: 'display'}
				]}
			/>
		);

		userEvent.click(screen.getByRole('group').children[0]); //clicking tab

		expect(handleClick).toHaveBeenCalled();
	});

Reply:
It seems that onClick isn't supplied to \<Tab>.
I think you found a bug.
It's very strange.The onClick has not been used anywhere, and I don't know how the tabGroup is actually being used.

I think Tab click behavior should be fixed, and it seems necessary to modify the related sampler and add tests. Could you create a CLM issue and see it as a very low priority? @daniel-stoian-lgp

Reply:
Sure, we'll create a CLM issue. Already on our TODO list